docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -93,6 +95,10 @@ The restore take a few seconds, after which expect to be logged out.
 
 You should be able to see that all of the data has been restored into the instance, including universes, users, metrics, alerts, task history, cloud providers, and so on.
 
+## Check results
+
+During a high-availability backup, the entire database is copied. If your universes are visible through YugabyteDB Anywhere UI and the replication timestamps are increasing, the backup is successful.
+
```

Comment:
More generally, does this page belong under Administration, being it's only (I think) about making the YB Anywhere highly available, not the deployments it is managing?

Reply:
Possibly. I don't remember placing this page under "manage deployments" -- it was either already there when I started at YB or someone from dev requested this page under "manage deployments". I'll think about it a bit before doing any restructuring.